content/en/profiler/enabling/supported_versions.md

```diff
@@ -9,46 +9,46 @@ further_reading:
 ---
```

Comment:
:arrow_up: Minor: Re: the page name, it's a bit weird using the term "tracer" here for the libraries, as in most other onboarding pages we use the term library or similar:

* https://docs.datadoghq.com/profiler/enabling/ => tracing libraries
* https://docs.datadoghq.com/tracing/trace_collection/custom_instrumentation/dd_libraries/ => datadog libraries
* https://docs.datadoghq.com/tracing/trace_collection/automatic_instrumentation/dd_libraries/ => tracing libraries

Reply:
Noted, that's good feedback. I updated the title and some of the other language to be more consistent with "tracing libraries".